client/app/telecom/pack/migration/service-delete/telecom-pack-migration-service-delete.controller.js

```diff
@@ -30,8 +30,22 @@ angular.module('managerApp').controller('TelecomPackMigrationServiceDeleteCtrl',
 
   function init() {
     self.process = TucPackMigrationProcess.getMigrationProcess();
-
-    self.chunkedSubServices = _.chunk(self.process.selectedOffer.subServicesToDelete, 2);
+    TucPackMigrationProcess.getServicesToDelete().then((servicesToDelete) => {
+      const subServicesToDelete = [];
+      angular.forEach(servicesToDelete, (subService) => {
```

Comment:
`forEach` + `push` === `map` :)

```suggestion
     const subServicesToDelete = servicesToDelete.map((subService) => {
```